lib/erb_lint/linters/space_in_html_tag.rb

```diff
@@ -0,0 +1,119 @@
+# frozen_string_literal: true
+
+module ERBLint
+  module Linters
+    # Detects extra or missing whitespace in html tags.
+    class SpaceInHtmlTag < Linter
+      include LinterRegistry
+
+      def offenses(processed_source)
+        offenses = []
+        processed_source.ast.descendants(:tag).each do |tag_node|
+          start_solidus, name, attributes, end_solidus = *tag_node
+
+          next_loc = name&.loc&.start || attributes&.loc&.start ||
+            end_solidus&.loc&.start || tag_node.loc.stop
+          if start_solidus
```

Comment:
For my curiosity what does `solidus` means?

Reply:
Solidus is the right word for the U+002F character (aka forward slash), it is used in html specs (https://www.w3.org/TR/2011/WD-html5-20110525/syntax.html#start-tags) so I started using it in the tokenizer and then everywhere else.

Reply:
Ah cool, TIL a new word